src/EA.Weee.Web/Areas/Producer/Controllers/ProducerController.cs

```diff
@@ -148,6 +149,7 @@ private async Task SetBreadcrumb(Guid organisationId, string activity)
         }
 
         [HttpGet]
+        [SmallProducerSubmissionSubmitted(Order = 2)]
         [SmallProducerSubmissionContext]
```

Comment:
@will-hope98  add the order = 1 to the SmallProducerSubmissionContext

Reply:
Done